graphdb/janus/src/main/java/org/apache/atlas/repository/graphdb/janus/AtlasElasticsearchQuery.java

```diff
@@ -336,5 +341,10 @@ public double getScore() {
             }
             return Double.parseDouble(String.valueOf(hit.get("_score")));
         }
+
+        @Override
+        public Map<String, List<String>> getHighLights() {
+            return (Map<String, List<String>>) this.hit.get("highlight");
```

Comment:
Add null check here. this.hit.get("highlight") might return null

Reply:
added